docs/version.md

```diff
@@ -314,13 +314,13 @@ const Tezos = new TezosToolkit(rpcUrl);
 
 Tezos.setWalletProvider(wallet)
 const signedW = await Tezos.wallet.signFailingNoop({
-arbitrary: char2Bytes("Hello World"),
+arbitrary: stringToBytes("Hello World"),
```

Comment:
might want to keep version.md as is, back in v17.2.0 release note char2Bytes was used